include/acvp/acvp_lcl.h

```diff
@@ -93,7 +93,7 @@
 #define ACVP_LOG_TRUNCATED_STR "...[truncated]\n"
 //This MUST be the length of the above screen (want to avoid calculating at runtime frequently)
 #define ACVP_LOG_TRUNCATED_STR_LEN 15
-#define ACVP_LOG_MAX_MSG_LEN 2048
+#define ACVP_LOG_MAX_MSG_LEN 1024000
```

Comment:
Was this something we wanted to keep? Will make info and verbose logging extremely long with showing 1MB of every vector set that comes through

Reply:
Nope, just fixed it.